core/Version.php

```diff
@@ -22,7 +22,7 @@ final class Version
      * The current Matomo version.
      * @var string
      */
-    public const VERSION = '5.6.0-b1';
+    public const VERSION = '5.6.0-b3';
```

Comment:
Keep -b2 if it hasn't been released.
```suggestion
    public const VERSION = '5.6.0-b2';
```

Reply:
✅ Done!